pyproject.toml

```diff
@@ -17,6 +17,7 @@ pymysql = "^1.1.1"
 aiomysql = "^0.2.0"
 openpyxl = "^3.1.5"
 pandas = "^2.2.3"
+asyncpg = "^0.30.0"
```

Comment:
Required for making postgres connection in jupyter notebook. Should I move it to dev dependencies?

Reply:
No, I think it's ok to leave it in the main dependencies. Anyone trying to connect a postgres db to test will need it...